examples/Multi_Armed_Bandit/bernoulli_multi_armed_bandit.py

```diff
@@ -61,6 +62,10 @@ def draw_arm(self, arm_index: int) -> bool:
 
 ### Setup
 
+# Setting a random seed for reproducibility.
+
+set_random_seed(42)
```

Comment:
Would be great if you could slightly clean the commit history - e.g. instead of having two commits adding and reverting the CHANGELOG entry, simply drop both of them.

Reply:
done :)